firebase-vscode/src/options.ts

```diff
@@ -12,16 +12,20 @@ import * as commandUtils from "../../src/emulator/commandUtils";
  * Passed to command.prepare()
  */
 
-interface CliOptions extends BaseOptions {
-  exportOnExit?: boolean | string
-  import?: string
+interface CliOptions extends Omit<BaseOptions, "config"> {
+  config: string;
+  exportOnExit?: boolean | string;
```

Comment:
I just realized, are these legit CLI options that are used in the CLI? If so maybe we can add it to BaseOptions and the CLI can benefit from this typing too?

Reply:
Good point - fixed